e2e/playwright/native-file-menu.spec.ts

```diff
@@ -438,6 +438,43 @@ test.describe('Native file menu', { tag: ['@electron'] }, () => {
         const expected = 'Open sample'
         expect(actual).toBe(expected)
       })
+      test('Modeling.File.Insert from project file', async ({
+        tronApp,
+        cmdBar,
+        page,
+        homePage,
+        scene,
+      }) => {
+        if (!tronApp) {
+          throwTronAppMissing()
+          return
+        }
```

Comment:
Shouldn't this be abstracted into one of the fixtures, and just be a oneLiner, or maybe part of "goToModelingScene"?

Reply:
I might leave this with @nadr0 as a separate issue if that's cool. This pattern only exists in `
e2e/playwright/native-file-menu.spec.ts`, not sure how much it should be generalized.

But I agree with the sentiment yes!